component/src/main/java/org/wso2/extension/siddhi/map/json/sourcemapper/JsonSourceMapper.java

```diff
@@ -464,7 +464,7 @@ private boolean isJsonValid(String jsonInString) {
 
     @Override
     public Class[] getSupportedInputEventClasses() {
-        return new Class[]{Event.class, Event[].class};
+        return new Class[]{String.class, Event.class, Event[].class};
```

Comment:
Why are you having Event.class, Event[].class ? 
JSON mapper only supports String right ?